api/src/pcapi/admin/base_configuration.py

```diff
@@ -129,40 +157,34 @@ def authorize(self):  # type: ignore [no-untyped-def]
         google_email = google_user["email"]
         db_user = users_repository.find_user_by_email(google_email)
         if db_user and not db_user.isActive:
-            return Forbidden()
+            return werkzeug.exceptions.Forbidden()
 
         if not db_user:
             if settings.IS_TESTING or settings.IS_DEV:
```

Comment:
(détail) peut-on arriver ici si `IS_DEV` est vrai ? J'ai l'impression que non (car la fonction "login_view" nous authentifiera directement) et donc qu'on pourrait supprimer cette partie de la condition.

```suggestion
            if settings.IS_TESTING:
```

Reply:
Bien vu, effectivement on arrivera jamais ici en environnement de dev.